utils/post_utils.jsx

```diff
@@ -93,6 +93,38 @@ export function canEditPost(post, editDisableAction) {
     return canEdit;
 }
 
+export function canRethreadPost(post, editDisableAction) {
+    if (isSystemMessage(post)) {
+        return false;
+    }
+
+    let canRethread = false;
+    let canRethreadOthers = false;
+    const license = getLicense(store.getState());
+    const config = getConfig(store.getState());
+    const channel = getChannel(store.getState(), post.channel_id);
+
+    const isOwner = isPostOwner(post);
+    canRethread = haveIChannelPermission(store.getState(), {channel: post.channel_id, team: channel && channel.team_id, permission: Permissions.EDIT_POST});
+    canRethreadOthers = haveIChannelPermission(store.getState(), {channel: post.channel_id, team: channel && channel.team_id, permission: Permissions.EDIT_OTHERS_POSTS});
+    if (!isOwner && !canRethreadOthers) {
+        canRethread = false;
+    }
+
+    if (canRethread && license.IsLicensed === 'true') {
```

Comment:
do edit time limits usually apply for editing posts of others? (if yes, isn't that a bug? i'd expect an admin who can edit posts of others to do that regardless of how old the post is)

Reply:
The time limit is a bit useless feature for rethreading. Do you think it should be removed entirely?